nuntium/templates/nuntium/instance_detail.html

```diff
@@ -24,10 +24,6 @@ <h2 class="instance-hero__heading">
                         {% include 'thread/message_mini.html' %}
                     {% endfor %}
                 </div>
-                <div class="instance-summary__popular">
```

Comment:
Use a Django `{% comment %}` so we remember where this should go when we put it back?